api/api.yaml

```diff
@@ -667,6 +667,13 @@ paths:
         Call this endpoint to notify the app of an update for one of its customers, indicating that an eOverdracht FHIR task was update or created.
         The customer is identified by the access-token.
       operationId: notifyTransferUpdate
+      parameters:
```

Comment:
Isn't this the endpoint that is called by the remote XIS to inform the local XIS of an updated task (the FHIR notification from the Bolt) (I figured this, since its an `/external` endpoint).

If so, the task ID should be a path parameter, rather than a query parameter (according to https://github.com/nuts-foundation/bolts/pull/16)

Reply:
Already fixed ;)